.gitignore

```diff
@@ -5,7 +5,7 @@ dist
 target
 packages/next/wasm/@next
 tarballs/
-packages/next/*.tgz
+packages/**/*.tgz
```

Comment:
For local testing. I don't think there's any drawbacks on this change?

Reply:
Yea seems fine